lighthouse-core/report/v2/report-template.html

```diff
@@ -21,14 +21,15 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1">
   <title>Lighthouse Report</title>
+  <style>%%LIGHTHOUSE_CSS%%</style>
```

Comment:
This one causes vscode's syntax highlighting to blow up. WDYT about making this replace str a css comment: <style>/*%%LIGHTHOUSE_CSS%%*/</style>?